frontend/src/contexts/AuthContext.js

```diff
@@ -2,12 +2,14 @@ import {
     getAuth,
     GoogleAuthProvider,
     onAuthStateChanged,
+    signInWithEmailAndPassword,
```

Comment:
doesn't seem like this is being used,, so we should remove it. can you create a new issue for implementing email sign-in for any non-watonomous emails that are in our whitelist?